apiary.apib

```diff
@@ -3315,6 +3317,151 @@ Get information about a meeting.
                     + open
                     + done
 
+## Calls [/calls]
+
+### calls.add [POST /calls.add]
+
+Add a new call.
+
++ Request (application/json)
+
+    + Attributes
+        + description: `Description of the call` (string, optional)
+        + customer (object, required)
+            + type (enum, required)
+                + Members
+                    + company
+                    + contact
+            + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
+        + due_date: `2016-02-04T16:00:00+00:00` (string, required)
+        + responsible_user_id: `98b2863e-7b01-4232-82f5-ede1f0b9db22` (string, required)
```

Comment:
Q: I'm wondering if we shouldn't allow to be open for change in the future where we would allow a team to be assigned, just like tasks?


```suggestion
        + responsible: `98b2863e-7b01-4232-82f5-ede1f0b9db22` (object, required)
            + type (enum, required)
                + Members
                    + user
            + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
```